concourse/pipelines/deploy.yml

```diff
@@ -618,7 +618,7 @@ jobs:
         ASSUME_ROLE_ARN: ((concourse_deployer_role_arn))
         APPLICATION: smokey
         VARIANT: default
-        COMMAND: bundle exec cucumber --profile test --strict-undefined -t @replatforming -t \"not @notreplatforming\"
+        COMMAND: bundle exec cucumber --profile test --strict-undefined -t @replatforming -t \"not @notreplatforming and not @replatformingnot((govuk_environment))\"
```

Comment:
Could we use `@not((govuk_environment))` instead since there's an existing `@notproduction` tag?

Reply:
We could, absolutely, especially if this `@replatforming` tag is going to go away?